.nf-core.yml

```diff
@@ -1,8 +1,22 @@
-repository_type: pipeline
-nf_core_version: "2.14.1"
+bump_version: null
 lint:
-  actions_ci: False
+  actions_ci: false
   files_exist:
     - lib/nfcore_external_java_deps.jar
     - conf/modules.config
   modules_config: false
+nf_core_version: 3.0.2
+org_path: null
+repository_type: pipeline
+template:
+  author: Phil Ewels
```

Comment:
Should we update author at some point, I feel @ewels hasn't been the main dev in this pipeline for a bit

Reply:
I will check with him on methylseq ch and resolve this in a future PR